cmd/memorydiskserver/memorydiskserver.go

```diff
@@ -63,6 +75,15 @@ func setHandler(w http.ResponseWriter, r *http.Request) {
 //}
 
 func main() {
+	// Read the starting value
+	dat, err := ioutil.ReadFile(STORE)
```

Comment:
I believe your first line will fail if the file already exists, which it will always do.